src/block/ChorusFlower.php

```diff
@@ -118,6 +119,12 @@ private function allHorizontalBlocksEmpty(World $world, Vector3 $position, ?int
 		return true;
 	}
 
+	public function onProjectileInteraction(Projectile $projectile) : void{
+		if($projectile instanceof WindCharge) {
+			$this->position->getWorld()->useBreakOn($this->position);
```

Comment:
From my tests Chorus Flowers aren't affected by this. They only break on direct projectile hit which is already covered. So, this is unnecessary.

Reply:
@ipad54 it seems to work on my machine (joke intended)

https://streamable.com/mtr10y

Reply:
nevermind you meant in vanilla 😆